doc/ca65.sgml

```diff
@@ -655,7 +655,54 @@ problem in most cases.
 
 To force a specific order of evaluation, parentheses may be used, as usual.
 
+<sect>String Constants and Raw-String Constants<label id="String_Constants"><p>
```

Comment:
It should be a `<sect1>`.  It should be placed between
`<sect1>Number format`
and
`<sect1>Conditional assembly`.

In other words, it's a companion of the "Number format" subsection.